tools/local-network-setup/.dh_origintrail_noderc

```diff
@@ -23,6 +23,7 @@
     ]
   },
   "ipWhitelist": [
+    "::1",
```

Comment:
@kotlarmilos without this line developers aren't available to shoot on to the nodes API, getting **Access denied** in local environment which is created with **setup-macos-environment.sh**